examples/deployment-manager/autogen/multivm/README.md

```diff
@@ -1,84 +1,73 @@
-values must be set for your particular solution.
+values.
+
+Set the variables for the Google Cloud project and name of the VM image:
 
 ```
 PROJECT_ID=<PROJECT_ID>
 IMAGE=redis
 ```
 
-After setting the variables above run the following commands to set the
-variables in `configurations.yaml`
+Next, run the following commands to update the values in `configurations.yaml`:
 
 ```bash
 mpdev cfg set redis/ projectId $PROJECT_ID
 mpdev cfg set redis/ image $IMAGE
 mpdev cfg set redis/ defaultReplicas 4
-
-# Optionally set deployment package zip output path.
-# Set to GCS location (e.g. gs://<BUCKET>/<OBJECT>) if using 
-# Producer Portal (Public Preview)
-mpdev cfg set redis/ zipPath template.zip
 ```
 
-Now generate a Deployment Manager template with the following command:
+### Generate the Deployment Manager template
+
+With the updated Autogen specification, run the following command to generate a Deployment Manager template:
 
 ```
 mpdev apply -f redis/configurations.yaml
 ```
 
-The template will be zipped to `redis/template.zip`, the location specified
+The template is zipped to `redis/template.zip`, which is the location specified
 in the `DeploymentManagerTemplate` resource of `redis/configurations.yaml`.
 
-To verify the template is properly configured, you can create a deployment from
-the template with the following commands:
+### Deploy the VM
+
+To verify the template is properly configured, create a deployment from
+the template:
 
 ```
 TMPDIR=$(mktemp -d)
 unzip redis/template.zip -d $TMPDIR
 gcloud deployment-manager deployments create redis --config $TMPDIR/test_config.yaml
 ```
 
-## Further Customization of Deployment Manager Template
-
-When creating your own Deployment Manager solution, you may need to
-customize other fields in the autogen specification, such as `passwords`,
-`deployInput`, and `postDeploy`. See the 
-[autogen reference](../../../../docs/autogen-reference.md)
-for explanations of the fields.
-
-## Upload solution to Partner Portal
-
-See instructions in this 
-[guide](../../../../docs/deployment-manager-guide.md#upload-solution-to-partner-portal).
-
-## Upload product to Producer Portal
-**Preview**: This product or feature is covered by the [Pre-GA Offerings Terms](https://cloud.devsite.corp.google.com/terms/service-terms#1) of the Google Cloud Platform Terms of Service. Pre-GA products and features may have limited support, and changes to pre-GA products and features may not be compatible with other pre-GA versions. For more information, see the [launch stage descriptions](https://cloud.devsite.corp.google.com/products#product-launch-stages).
+## What's next
 
-See instructions in this
-[guide](../../../../docs/deployment-manager-guide.md#upload-product-to-producer-portal)
+See [Creating your deployment package](https://cloud.google.com/marketplace/docs/partners/vm/create-deployment-package) to learn how to learn how to generate Deployment Manager templates for your VM product.
```

Comment:
Should we link to the section specifically about submitting the deployment package?

Reply:
I think it's better to have them start at the beginning of the doc to get all the context- Also there are options between portals so it seems better to keep it high-level. My goal now for these readme files is that they review an example (and all the related commands) but to create a package on their own, they should use the other how-to doc as the main source of truth (rather than jump between sections of two separate docs). HTH - let me know if you have any concerns.